.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: c
```

Comment:
No dependency, besides docker, is required on the host. I'd suggest to use:

```
language: minimal
install: skip
```

There should be no practical difference, but it makes it explicit.